Makefile

```diff
@@ -16,9 +16,10 @@ build-and-push-api: build-api-image push-api-image
 build-and-push-resource-processor: build-resource-processor-vm-porter-image push-resource-processor-vm-porter-image
 build-and-push-gitea: build-gitea-image push-gitea-image
 build-and-push-guacamole: build-guacamole-image push-guacamole-image
+build-and-push-firewall: build-firewall-bundle push-firewall-bundle
```

Comment:
can't see where these targets actually are - did they get removed and this left behind?

Reply:
Good catch, yes these aren't used anymore